v23.1/take-and-restore-encrypted-backups.md

```diff
@@ -60,6 +61,50 @@ The AWS URI **requires** the following:
 
 See AWS's [KMS keys](https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html) documentation for guidance on creating an AWS KMS key.
 
+#### Azure Key Vault URI format
+
+{% include_cached new-in.html version="v23.1" %} The Azure Key Vault URI must use the following format:
+
+~~~
+azure-kms:///{key}/{key version}?AZURE_TENANT_ID={tenant ID}&AZURE_CLIENT_ID={client ID}&AZURE_CLIENT_SECRET={client secret}&AZURE_VAULT_NAME={key vault name}
+~~~
+
+To use `implicit` authentication for KMS URI for an Azure encrypted backup, include the Azure Key Vault name:
```

Comment:
Eng review: Is this correct here? I gathered that a user could specify `implicit` auth in the KMS URI too, but not 100% on this?